### PR TITLE
Classify multiple bugs at the same time #1866

### PR DESCRIPTION
**Pull request by willT97**
Hello there this is my first contribution.

I have modified the code to classify the bugs at the same time by appending them to a list.
I then iterate over that list and apply the same logic as before.

### CODE REVIEW

**Comment by willT97** [comment]
For the issue #1866

File: scripts/regressor_finder.py [hunk 1]
```diff
@@ -262,7 +263,15 @@ def append_bug_fixing_commits(bug_id: int, type_: str) -> None:
...
                 continue
 
-            if defect_model.classify(bug)[0] == "defect":
+            bugs_to_classify.append(bug)
+
+        classified_bugs = []
+        if bugs_to_classify:
+            classified_bugs = defect_model.classify(bugs_to_classify)
+
+        for i, label in enumerate(classified_bugs):
```
Comment (marco-c) [inline scripts/regressor_finder.py:272]: Nit: you can do `for i, (bug, label) in enumerate(zip(bugs_to_classify, classified_bugs)):`
so you can skip `bug = bugs_to_classify[i]`

Comment (willT97) [reply scripts/regressor_finder.py]: Thanks, I only use i for getting the bug so just removed the enumerate and kept the zip.

Comment (marco-c) [reply scripts/regressor_finder.py]: Good idea

**Review by marco-c — APPROVED** [review]

**Comment by marco-c** [comment]
Thanks!

**Pull request merged**